### PR TITLE
@craigspaeth: added associated articles for fairs

### PR DESCRIPTION
**Pull request by xtina-starr**
This is for the new associated articles for the fair info app in force. Related to [3364](https://github.com/artsy/force/issues/3364), [3365](https://github.com/artsy/force/issues/3365), [3366](https://github.com/artsy/force/issues/3366)

cc: @kanaabe 

screen shot 2015-12-08 at 2 42 51 pm

### CODE REVIEW

**Comment by kanaabe** [comment]
@craigspaeth we should probably merge your PR first with the simple-modal updates, pull those changes, then reinstall node modules here using node 5. was npm updated at all in the build?

**Comment by craigspaeth** [comment]
@kanaabe Nah I'll deal with that later. I'm going to set Semaphore back and see if we can get the passing + merged first.

File: api/apps/articles/test/model.coffee [hunk 1]
```diff
@@ -142,6 +142,45 @@ describe 'Article', ->
             done()
         )
 
+    it 'can return articles by a fair programming id', (done) ->
```
Comment (craigspaeth) [inline api/apps/articles/test/model.coffee]: Nice set of tests!

File: client/apps/edit/components/admin/index.coffee [hunk 2]
```diff
@@ -60,6 +63,81 @@ module.exports = class EditAdmin extends Backbone.View
...
+          .get("#{sd.ARTSY_URL}/api/v1/fair/#{id}")
+          .set('X-Access-Token': sd.USER.access_token).end (err, res) =>
+            @fairs.push id: res.body._id, value: res.body.name
+            cb()
+      , =>
+        list.setState loading: false, items: @fairs
+    else
+      list.setState loading: false
+
+  setupAboutTheFairAutocomplete: ->
```
Comment (craigspaeth) [inline client/apps/edit/components/admin/index.coffee]: Looks good! We might want to think about breaking this admin view into multiple parts now (maybe a view per-admin section e.g. AdminFeaturingView, AdminMetadataView). Also probably an opportunity to DRY up this autocomplete init code, but that one is actually less offensive to me b/c it's mostly config.

**Comment by craigspaeth** [comment]
:clap: well done! No complaints! Great work pairing on this @kanaabe!

**Pull request merged**